tasks/task880_schema_guided_dstc8_classification.json

```diff
@@ -0,0 +1,8106 @@
+{
+    "Contributors": ["Subba Raja Kashyap Saligrama"],
+    "Source": ["schema_guided_dstc8 (https://huggingface.co/datasets/schema_guided_dstc8)"],
+    "Categories": ["Classification"],
+    "Definition": "You are given a sentence from a conversation between a human and a virtual assistant. Your task is to classify the sentence into one of following five action categories - INFORM, INFORM_INTENT, OFFER, REQUEST, REQUEST_ALTS",
```

Comment:
I am not sure if the definition is clear. 

How are `INFORM` and `INFORM_INTENT`? 
Similarly, `REQUEST` and  `REQUEST_ALTS`? (what does `ALTS` stand for?)